documents/operating.md

```diff
@@ -281,49 +284,79 @@ Invoices that are not approved by a project's Comptroller will not be compensate
+### Paying Back BIFFUD
+If a project that had been allocated money from BIFFUD has any gross income, that project's coffers (not including the original allocation) shall be instantly deducted until the original allocation is offset in full.  In other words, project owners should pay back BIFFUD before paying themselves.
+
+BIF Tithe and BIF Tax amounts do not count towards paying back BIFFUD.
+
+### Interproject Loans
+An interproject fund transfer can be specified as a loan.  This would result in similar processes to the one described in "Paying Back BIFFUD."  All loans are to be paid back in the order they were received.
+
+In other words, BIFFUD gets paid back first, then projects get paid back in the order that loans were made.
+
+### Corporate Coffers and Spoils of War
+"Corporate Coffers" represent all undistributed corporate capital.
+
+At the end of a fiscal year the net of all income and expense that is not associated with projects with owners (e.g. non-corporate projects) combined with all Tithe, Overhead, Grown up Pants, and Voluntary Contributions shall be added to the Corporate Coffers.
+
+The net change to the Corporate Coffers (aka the profit or loss) in a given year shall be called "The Spoils of War" and shall be allocated to
+
+### Payment Cadence
+Everybody will be paid what they are due at whatever cadence is agreed upon between the relevant parties and also supported by processes agreed upon by the BIFFUD Operating Branch.
+
+### Targeted Allocations
+For Tax purposes, the following outlines explicitly how capital is to be allocated to Corporate Overlords in a given fiscal year with the following priority:
+
+1. Corporate overlords are allocated any portion of Project Coffers that has been associated to them as described in the "Distributing Project Money" section.
+
+2. Corporate overlords are allocated any portion of the Spoils of War according to their pro-rated ownership percentage for that year.
+
+Put another way, Corporate Overlords are (1) entitled to a portion of relevant Project Coffers according to whatever has been determined within those projects' governance structures and then also (2) entitled to a percentage of corporate profit according to their pro-rated ownership percentage.
+
+Project Owners who are not Corporate Overlords are still liable (from tax purposes) for whatever portions of Project Coffers that they are entitled to.
+
+### Distributing Corporate Capital
+A mandatory distribution will be made by May 31st to offset individual taxes owed by Corporate Overlords as a result of The Spoils of War.  Corporate Overlords may vote to prevent this distribution, as described in Section 5.  Corporate overlords must invoice BIFFUD with proof of their tax liability in order to be eligable for the reimbursement.
+
+Corporate Overlords may vote at any time to distribute capital from Corporate Coffers in the Corporate Coffers in proportion of their net corporate capital positions.
```

Comment:
Is this a repetition of "Corporate Coffers"?

Reply:
It is indeed -- will fix the typo before merging.